plat/linuxu/Config.uk

```diff
@@ -1,7 +1,7 @@
 menuconfig PLAT_LINUXU
        bool "Linux user space"
        default n
-       depends on !HAVE_SYSCALL && !HAVE_SMP && ((!ARCH_ARM_32 && !ARCH_ARM_64) || !HAVE_SCHED)
+       depends on !HAVE_SYSCALL && !HAVE_SMP && (!ARCH_ARM_32 || !HAVE_SCHED)
```

Comment:
Why is `ARCH_ARM64` removed?

Reply:
Because with the changes, the linux user space target finally works also for Aarch64. It is making `linuxu` available when Arm is selected under architecture configuration.